### DISCUSSION TITLE
First barrier: `jj split` sends me to a TUI that I don't know how to complete

### DISCUSSION THREAD

**Discussion by canadaduane** [jj-vcs/jj#3714] [Ideas]
First time user here.

I used `jj split` and it took me to a TUI where I marked some of the files as belonging to the new commit. I can't get out, except by canceling? I press ESC, Ctrl-C, or "Q" and it lets me quit, but I don't want to quit, I want to accept the marked files.

**Comment by thoughtpolice** [comment]
Sorry for the delay in reply; thanks for this report. Unfortunately, this is something a ton of new users are tripping on, and so you're not alone. In general, the keybinds aren't discoverable.

In your case, you can hit `c` (for "commit") in order to put the changes you selected into a commit, and all other changes will get put into another commit on top of that. Also note that the "menu" in the TUI is actually mouse-aware, so you can click on the "Help" button with your mouse to get a keybind popup.

We've gotten enough of these reports from new users that this is clearly something we can't ignore. I'm going to spend some time trying to fix this later this week. Thanks.

**Reply by canadaduane** [reply]
This helps a bunch, thanks!
